### ISSUE TITLE
warning  `>` can be escaped with `&gt;`

### ISSUE THREAD

**Issue by chenxsan** [mdx-js/eslint-mdx#287]
### Subject of the issue

Here's the content of a mdx file:

```mdx
Webpack 5 has been officially released. Read our <a href="/blog/2020-10-10-webpack-5-release/">announcement</a>. Not ready yet? Read <a href="https://v4.webpack.js.org/">webpack 4 documentation here</a>.
```

And the eslint configuration:

```js
module.exports = {
  root: true,
  extends: ['eslint:recommended', 'plugin:react/recommended', 'prettier'],
  parser: '@babel/eslint-parser',
  env: {
    browser: true,
    es6: true,
    node: true,
    jest: true,
    'cypress/globals': true,
  },
  plugins: ['markdown', 'cypress', 'react-hooks'],
  globals: {
    __DEV__: true,
  },
  rules: {
    'no-console': 'off',
    semi: ['error', 'always'],
    quotes: ['error', 'single'],
    'react/jsx-uses-react': 'off', // no longer needed with new jsx transform
    'react/react-in-jsx-scope': 'off', // ditto
    'react-hooks/rules-of-hooks': 'error',
    'react-hooks/exhaustive-deps': 'warn',
  },
  settings: {
    react: {
      version: 'detect',
    },
  },
  overrides: [
    { files: ['src/**/*.jsx'] }, // eslint would lint .js only by default
    {
      files: ['**/*.mdx'],
      extends: ['plugin:mdx/recommended'],
      globals: {
        Badge: true,
      },
      rules: {
        semi: ['off'],
      },
      settings: {
        'mdx/code-blocks': true,
      },
    },
    {
      files: ['**/*.md'],
      processor: 'markdown/markdown',
    },
    {
      files: ['**/*.{md,mdx}/*.{js,javascript}'], // we don't lint ts at the moment
      rules: {
        indent: ['error', 2],
        quotes: ['error', 'single'],
        'no-undef': 'off',
        'no-unused-vars': 'off',
        'no-constant-condition': 'off',
        'no-useless-escape': 'off',
        'no-dupe-keys': 'off',
      },
    },
  ],
};
```
And here's the lint result:

```bash
  1:94   warning  `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`  mdx/no-unescaped-entities
  1:95   warning  `>` can be escaped with `&gt;`                                   mdx/no-unescaped-entities
  1:111  warning  `>` can be escaped with `&gt;`                                   mdx/no-unescaped-entities
```

### Your environment

*   **OS**: 
*   **Packages**: 
*   **Env**: 

### Steps to reproduce

Here's the pull request https://github.com/webpack/webpack.js.org/pull/4326 you can use to reproduce:

1. clone the pull request
2. run `yarn` to install
3. run `yarn lint:js`

### Expected behaviour

No warning showed.

### Actual behaviour

Warnings.
